libraries/ms-common/inc/gpio_it.h

```diff
@@ -4,7 +4,7 @@
 #include <stdint.h>
 
 #include "gpio.h"
-#include "interrupt.h"
+#include "interrupt_def.h"
```

Comment:
I don't think we need to explicitly include interrupt_def.h? It should be included with stm32f0xx_interrupt.h